plugins/scaffolder/src/components/fields/RepoUrlPicker/validation.ts

```diff
@@ -21,7 +21,7 @@ export const repoPickerValidation = (
 ) => {
   try {
     const { host, searchParams } = new URL(`https://${value}`);
-    if (!host || !searchParams.get('owner') || !searchParams.get('repo')) {
+    if (!host || !searchParams.get('repo')) {
```

Comment:
@Rugvip need some help here please, how do I get the integrationApi here? I need that to determine the type via host again for validation.